pkg/live/inventoryrg.go

```diff
@@ -79,14 +97,14 @@ func WrapInventoryObj(obj *unstructured.Unstructured) inventory.Storage {
 	if obj != nil {
 		klog.V(4).Infof("wrapping Inventory obj: %s/%s\n", obj.GetNamespace(), obj.GetName())
 	}
-	return &InventoryResourceGroup{inv: obj}
+	return &InventoryResourceGroup{inv: obj, strategy: inventory.NameStrategy}
```

Comment:
kpt has only supported name strategy, and this PR doesn't change that behaviour, as we used to always hardcode strategy to Name. I'm taking advantage of the label strategy for destroy commands to allow deleting all sharded ResourceGroups when the command is run. Added a function comment to document this. We only need it as a function argument for WrapInventoryInfoObj as that is the one caller in the destroy command that needs it to be of label strategy.

Reply:
The `WrapInventoryObj` func satisfies an interface required to pass it to `inventory.NewClient`. 

I thought `WrapInventoryInfoObj` also used to satisfy an interface for a different inventory function. Does it not?

If `WrapInventoryInfoObj` is only used in `ToInventoryInfo` could `wrapInventoryInfoObj` be a private function?

Reply:
`WrapInventoryInfoObj` does not need to satisfy interface requirements unlike `WrapInventoryObj`. The function is still an exported public function since it is used in the live migrate command `internal/cmdmigrate.go`.